pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -2194,6 +2194,60 @@ func IsResourceTypePVC(backup *stork_api.ApplicationBackup) bool {
 func (a *ApplicationBackupController) cleanupResources(
 	backup *stork_api.ApplicationBackup,
 ) error {
+	if IsBackupObjectTypeVirtualMachine(backup) && !backup.Spec.SkipAutoExecRules {
```

Comment:
Missed checking !skipVMAutoExecRule?

Reply:
@vsundarraj-px do you mean `backup.Spec.SkipAutoExecRules` ? I could not see any `skipVMAutoExecRule`